webpack.config.js

```diff
@@ -32,11 +23,79 @@ module.exports = {
         // exclude: /node_modules/,
         loader: "babel-loader",
       },
+      {
+        test: /\.css$/,
+        use: [MiniCssExtractPlugin.loader, "css-loader"],
+      },
```

Comment:
I don't think we need a css loader right now. I think you can strike this and all the MiniCssExtractPlugin stuff.

Reply:
I tried removing all that (and tried it before adding it in) and it wasn't properly including the css. If you can show me how to do it, I'm down.

Reply:
OK. I'll take a look tonight.